### PR TITLE
Map char replacement in email addresses

### PR DESCRIPTION
**Pull request by alexbourret**

### CODE REVIEW

File: python-lib/azure_client.py [hunk 1]
```diff
@@ -69,6 +72,17 @@ def __init__(self, project_key, config):
         # Connect to Graph API
         self.set_session_headers()
 
+    def assert_valid_login_remapping(self):
+        for login_remapping in self.login_remapping:
+            print("ALX:login_remapping={}".format(login_remapping))
```
Comment (StanislasGuinel) [inline python-lib/azure_client.py:77]: remove the print()

File: python-lib/azure_client.py [hunk 2]
```diff
@@ -69,6 +72,17 @@ def __init__(self, project_key, config):
...
+            to_value = login_remapping.get("to")
+            if to_value and not self.is_valid_login(to_value):
+                raise Exception("'{}' in the login remapping is not a valid character for a DSS user login".format(to_value))
```
Comment (StanislasGuinel) [inline python-lib/azure_client.py:80]: maybe add the allowed characters to the message, for example "valid characters must match the regex pattern [a-zA-Z0-9@.+_-]"

**Review by StanislasGuinel — COMMENTED** [review]
No bug found after testing.

File: python-lib/azure_client.py (hunk 1 above)
Comment (alexbourret) [reply python-lib/azure_client.py:77]: lol

File: python-lib/azure_client.py (hunk 2 above)
Comment (alexbourret) [reply python-lib/azure_client.py:80]: 👍

**Review by StanislasGuinel — APPROVED** [review]

**Pull request merged**